main.go

```diff
@@ -1,7 +1,34 @@
 package main
 
-import "fmt"
+import (
+	"encoding/json"
+	"io"
+	"log"
+	"net/http"
+	"time"
+)
 
 func main() {
-	fmt.Println("Dupa")
+	client := &http.Client{}
```

Comment:
Warto tworzyć więcej ustawień clienta np. jakiś timeout, a najfajniej mieć jakąś jedną instancję którą reużywasz w całej apce.

Reply:
Cała apka siedzi w mainie, jest sens to wydzielać?